### ISSUE TITLE
Presidium Prophet quest can be uncompletable if you don't do it immediately

### ISSUE THREAD

**Issue by henbagle** [henbagle/LE1CommunityPatch#28]
**Describe the bug**
Presidium Prophet quest can be uncompletable if you convince either C-Sec/Hanar and then leave the area without speaking to the other, as they will disappear forever once you leave

**Other Considerations**
This bug was fixed in ME1Recalibrated, so a fix can probably be ported over

**Comment by henbagle** [comment]
The fix in ME1Recalibrated used a series of PMCheckStates in addition to the vanilla conditionals to trigger the visibility of the C-SEC Officer and the Hanar. A better solution would be to update the conditionals using the PlotManagerUpdate feature.

To the best of my knowledge, here are the three outcomes from leaving the area after persuading/purchasing the license.
1. Persuade the officer to let the hanar preach. Need to talk to hanar. Leave area, come back, hanar is still there, C-SEC officer is gone. No bugs.
2. Purchase the license. Hanar goes away, need to talk to officer. Leave area, come back. The hanar is there and talks as if you have convinced the officer to go away. Bug.
3. Persuade the Hanar to go away, need to talk to officer. Leave area, come back. Nobody is there and you can't complete the mission.

Still investigating a fix for outcome 2, but outcome three can be fixed by updating conditional 1269 (Is Cop Present) to the following. Bool 4748 (Ended Argument) changed to 4752 (Completed Plot)
```public function bool F1269(BioWorldInfo bioWorld, int Argument)
{
    local BioGlobalVariableTable gv;
    
    gv = bioWorld.GetGlobalVariables();
    return (gv.GetBool(4752) == FALSE && gv.GetBool(4751) == FALSE) && gv.GetBool(4398) == TRUE;
}
```

**Comment by henbagle** [comment]
EDIT: Conditional F1274 has been updated to fix #110. Original comment has been edited with updated conditional text
